### DISCUSSION TITLE
v0.46.1

### DISCUSSION THREAD

**Discussion by wata727** [terraform-linters/tflint#1748] [Announcements]
### BugFixes

- [#1746](https://github.com/terraform-linters/tflint/pull/1746): sarif: fix incorrect `endLine` ([@tbutler-qontigo](https://github.com/tbutler-qontigo))

### Chores

- [#1738](https://github.com/terraform-linters/tflint/pull/1738): build(deps): Bump sigstore/cosign-installer from 3.0.1 to 3.0.2
- [#1739](https://github.com/terraform-linters/tflint/pull/1739): build(deps): Bump golang.org/x/crypto from 0.7.0 to 0.8.0
- [#1741](https://github.com/terraform-linters/tflint/pull/1741): build(deps): Bump golang.org/x/oauth2 from 0.6.0 to 0.7.0
- [#1743](https://github.com/terraform-linters/tflint/pull/1743): build(deps): Bump github.com/terraform-linters/tflint-plugin-sdk from 0.16.0 to 0.16.1

<hr /><em>This discussion was created from the release <a href='https://github.com/terraform-linters/tflint/releases/tag/v0.46.1'>v0.46.1</a>.</em>

**Comment by ChiefAntiphates** [comment]
Hi do you know when the next release is going to happen? There's some functionality that was released in commit [1dcdcc4f24c0af48908ea2ddfaa799d003309213](https://github.com/terraform-linters/tflint/commit/1dcdcc4f24c0af48908ea2ddfaa799d003309213) and I was hoping to use this without building from source.

Thanks
